### ISSUE TITLE
Include an example of the failing procedure spec

### ISSUE THREAD

**Issue by vincentmorneau** [OraOpenSource/Logger#206]
What if in logger extra column, we would provide the code to invoke the procedure/function with the current parameters.

![image](https://user-images.githubusercontent.com/5391291/36160932-72c7aa2a-10b0-11e8-8ee0-41ce5d401770.png)

Then, to try the procedure again programmatically (as we test the new changes), we just have to copy paste this into sql dev.

I often find myself typing the code manually, so it would be useful. Maybe a different column, I'm not sure.

**Comment by eaolson** [comment]
Is it even possible to get the names of the parameters to a function like this in PL/SQL, let alone the current values?

**Comment by vincentmorneau** [comment]
Yes, that is what Logger is doing in the example above. The parameters are logged and their values as well.

What this ticket is proposing is to format the output in a way that we could copy paste the code and run the procedure again.

**Comment by SvenWeller** [comment]
While I like the base concept, I think it is very situational. 
Meaning this will work only in some specific cases. For example it is not possible to run modules that do exist only in the package body and not in the spec. 

Also it assumes that the name of the packaged proc/function that is stored in the logger params does indeed match the name of the real proc/function that called logger.
